File: 3289-the-two-sneaky-numbers-of-digitville/3289-the-two-sneaky-numbers-of-digitville.cpp
class Solution {
public:
    vector<int> getSneakyNumbers(vector<int>& a) {
        int n=a.size();
        vector<int>ans;
        unordered_map<int,int>m;
        for(int i=0;i<n;i++){
            if(m.find(a[i])==m.end()){
                m[a[i]]=1;
            }
            else {
                ans.push_back(a[i]);
            }
        }
        sort(ans.begin(),ans.end());
        return ans;
    }
};

File: 3289-the-two-sneaky-numbers-of-digitville/3289-the-two-sneaky-numbers-of-digitville.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var getSneakyNumbers = function(nums) {
    let trace={};
    let ans=[];
    for(let i=0;i<nums.length;i++){
        trace[nums[i]] = (trace[nums[i]] || 0) + 1;
        if(trace[nums[i]]>1){
            ans.push(nums[i]);
        }
    }
    return ans;
};